Services/UI/OJS.Services.Ui.Data/Implementations/ContestsDataService.cs

```diff
@@ -60,7 +75,7 @@ public async Task<IEnumerable<TServiceModel>> GetAllPast<TServiceModel>()
                 .FirstOrDefaultAsync();
 
         public IQueryable<Contest> GetAllActive()
```

Comment:
All these are still `public` but `IQueryable`

Reply:
They are not part of this PR. Also they are not used anywhere, so they will be removed in future cleanup.